ibm/resource_ibm_database.go

```diff
@@ -678,6 +709,189 @@ type Params struct {
 	PITRTimeStamp       string `json:"point_in_time_recovery_time,omitempty"`
 }
 
+func getDatabaseServiceDefaults(service string, meta interface{}) (*icdv4.Group, error) {
+	icdClient, err := meta.(ClientSession).ICDAPI()
+	if err != nil {
+		return nil, fmt.Errorf("Error getting database client settings: %s", err)
+	}
+
+	var dbType string
+	if strings.HasPrefix(service, "messages-for-") {
+		dbType = service[len("messages-for-"):]
+		dbType = service[len("messages-for-"):]
+	} else {
+		dbType = service[len("databases-for-"):]
+	}
+
+	groupDefaults, err := icdClient.Groups().GetDefaultGroups(dbType)
+	if err != nil {
+		return nil, fmt.Errorf("ICD API is down for plan validation, set plan_validation=false")
+	}
```

Comment:
If the ICD deployables API fails, then let user know how to disable this feature to continue doing a deployment.  This is an edge case that should be needed, but until the deployables is working for all databases decided to have this feature.